eclipse-che4z/che-che4z-lsp-for-cobol
# fix: Skip analysis for not used copybook

## How Has This Been Tested?

1. Open program `A` that refers to `copybook A`
2. Open 'copybook A' and add an error
3. Ensure that program `A` was reanalyzed
4. Create a new COBOL file
5. Modify the new COBOL file
6. Ensure program `A` was not analyzed

1. Open program `A` that refers to `copybook A`
2. Open program `B` that refers to `copybook B`
3. Open program `C` that refers to `copybook A` and `copybook B`
4. Open 'copybook A' and add an error
5. Ensure that program `A` and program `C` was reanalyzed
6. Ensure that program `B` was not reanalyzed

1. Open program `D` that refers to `copybook D`
2. Open copybook `D` that refers to `copybook A`
3. Open copybook `A` and add an error
4. Ensure that program `A` was reanalyzed
5. Ensure that program `A`, `copybook D` and `copybook A` have errors


## Checklist:
